recipes/progress_reporter/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "progress_reporter" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }} 
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/marscher/progress_reporter
+  git_rev: {{ 'v' + version }}
```

Comment:
We prefer to use tarballs instead of cloning from SCM. This makes things faster and lighter weight for the CIs and any associated builds.

Please change this to use a tarball and include a checksum. Preferably `sha256`. Please see the [`example`]( https://github.com/conda-forge/staged-recipes/blob/327fa546004248e5a119b6afd2edd5391e967a04/recipes/example/meta.yaml ) for details.